apps/sms/js/recipients.js

```diff
@@ -184,6 +202,8 @@
     data.set(this, list);
     events.set(this, { add: [], remove: [], modechange: [] });
     view.set(this, new Recipients.View(this, setup));
+
+    this.validator = RecipientValidator.for(this);
```

Comment:
I looked at them and none seemed really good...

But now that I learnt about Symbols I could use that instead ;)